src/system/documents/chat-message.ts

```diff
@@ -261,8 +261,7 @@ export class CosmereChatMessage extends ChatMessage {
             const tooltipNormal = $(await rollNormal.getTooltip());
             this.enrichDamageTooltip(rollNormal, type, tooltipNormal);
             tooltipNormalHTML +=
-                tooltipNormal.find('.tooltip-part')[0].outerHTML;
-
+                tooltipNormal.find('.tooltip-part')[0]?.outerHTML || ``;
```

Comment:
this empty string alternative is a clutch.
It works, but the tooltips are no longer consistent. Screenshot:
![image](https://github.com/user-attachments/assets/ad0e00eb-ac50-41cb-89bc-23ed625edca4)

Reply:
Hmm this is annoying yeah. The problem is that the core foundry code only generates tooltips for dice. To show constants I do enrichment AFTER the tooltip is generated but that requires an already existing tooltip which obviously doesn't exist here. Not sure what the best fix would be unfortunately, i'd have to have a proper think about even more complicated tooltip generation code. I'll take a look at what the dnd5e system does in this case to get ideas